draft-ietf-capport-architecture.xml

```diff
@@ -640,6 +640,13 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
                 link. In such cases, identification could be performed by subnet, such as the /64
                 to which the IP belongs.
           </t>
+
+          <t>
+            Further attention should be paid to a device using dual-stack <xref target="RFC4213"/>:
+            it could have both an IPv4 and an IPv6 address at the same time. There could be no
+            properties in common between the two addresses, meaning that some form of mapping solution
+            could be required to form a single identity from the two address.
```

Comment:
addresses.

Reply:
I think dual-stack is covered in the above paragraph about multiple IP addresses. One IPv4 and one IPv6 is multiple addresses.

Reply:
How about we just change the previous paragraph to, "... may have multiple IP addresses (either IPv4 or IPv6), ..."

Reply:
We can maybe combine the two. The point was the mention the dual stack case, because it provides challenges beyond just having multiple IPv6 addresses (a reviewer explicitly asked for it to be called out). But, now that I think about it, that challenge already existed for ipv4 (as we well-know).  I want to make it clear that they can be assigned at the same time, though. I think reference dual-stack helps with that.

So, how about:

"may have multiple IP Addresses (either IPv4, IPv6 or a combination as in dual-stack)..."

Reply:
Sounds good. I think the challenges are the same for multiple IP addresses of any kind.
What do you think of the more concise, "either IPv4, IPv6, or a dual-stack combination" ?

Reply:
like it. I'll make the change. Thanks.